js/form.js

```diff
@@ -0,0 +1,132 @@
+import { initializeApp } from "https://www.gstatic.com/firebasejs/10.1.0/firebase-app.js";
+import { getDatabase, onValue, ref, set, get, child } from "https://www.gstatic.com/firebasejs/10.1.0/firebase-database.js";
+
+var firebaseConfig = {
```

Comment:
please move firebase configurations to a shared file called "service.js"

Reply:
done